app/Mail/GeneralInterestInvite.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class GeneralInterestInvite extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        $this->from('noreply@my.robojackets.org', 'RoboJackets')
```

Comment:
There's a default configuration in .env that is correct for most mail, but I want to send GI invites as "RoboJackets" not "MyRoboJackets".